docs/en/connector-v2/sink/SelectDB-Cloud.md

```diff
@@ -0,0 +1,120 @@
+# SelectDB Cloud
+
+> SelectDB Cloud sink connector
+
+## Description
+Used to send data to SelectDB Cloud. Both support streaming and batch mode.
+The internal implementation of SelectDB Cloud sink connector upload after batch caching and commit the CopyInto sql to load data into the table.
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+
+## Options
+
+| name                        | type   | required | default value   |
+|-----------------------------|--------|----------|-----------------|
+| load-url                    | string | yes      | -               |
+| jdbc-url                    | string | yes      | -               |
+| cluster-name                | string | yes      | -               |
+| username                    | string | yes      | -               |
+| password                    | string | yes      | -               |
+| table.identifier            | string | yes      | -               |
+| sink.properties             | string | yes      | -               |
```

Comment:
Change config prefix may be better.
Please refer : https://github.com/apache/incubator-seatunnel/pull/3856